docs/source/v1/WaterCycle/simulation_data/simulation_table.rst

```diff
@@ -0,0 +1,111 @@
+**********************************
```

Comment:
This page can be seen in rendered form at https://portal.nersc.gov/cfs/e3sm/forsyth/data_docs_59/html/v1/WaterCycle/simulation_data/simulation_table.html.